tide_landing_page.info.yml

```diff
@@ -44,6 +44,8 @@ config_devel:
     - core.entity_form_display.paragraph.card_navigation_featured_auto.default
     - core.entity_form_display.paragraph.card_promotion.default
     - core.entity_form_display.paragraph.card_promotion_auto.default
+    - core.entity_form_display.paragraph.embedded_search_form.default
+    - core.entity_view_display.paragraph.embedded_search_form.default
```

Comment:
We should maintain alphabetical order for these values, it helps maintainability.